examples/assets_pandas_type_metadata/assets_pandas_type_metadata/resources/csv_io_manager.py

```diff
@@ -29,26 +30,26 @@ def handle_output(self, context, obj: pd.DataFrame):
         with open(fpath + ".version", "w", encoding="utf8") as f:
             f.write(context.version if context.version else "None")
 
-        yield MetadataEntry.int(obj.shape[0], "Rows")
-        yield MetadataEntry.path(fpath, "Path")
-        yield MetadataEntry.md(obj.head(5).to_markdown(), "Sample")
-        yield MetadataEntry.text(context.version, "Resolved version")
-        yield MetadataEntry.table_schema(
-            self.get_schema(context.dagster_type),
+        yield MetadataEntry("Rows", value=MetadataValue.int(obj.shape[0]))
```

Comment:
Can we switch this to a dictionary? I think we generally don't want users to be constructing `MetadataEntry`s at all, right?

Reply:
done